src/test/mochitest/examples/long.js

```diff
@@ -5,66 +5,68 @@ var app = {};
 // may not even be worth separating this logic
 // out, but we do this to demonstrate one way to
 // separate out parts of your application.
-app.TodoModel = function (key) {
+app.TodoModel = function(key) {
   this.key = key;
   this.todos = [];
   this.onChanges = [];
 };
 
-app.TodoModel.prototype.addTodo = function (title) {
-  this.todos = this.todos.concat([{
-	id: Utils.uuid(),
-	title: title,
-	completed: false
-  }]);
+app.TodoModel.prototype.addTodo = function(title) {
+  this.todos = this.todos.concat([
+    {
+      id: Utils.uuid(),
+      title: title,
+      completed: false
+    }
+  ]);
 };
 
 app.TodoModel.prototype.inform = function() {
   // Something changed, but we do nothing
   return null;
 };
 
-app.TodoModel.prototype.toggleAll = function (checked) {
+app.TodoModel.prototype.toggleAll = function(checked) {
   // Note: it's usually better to use immutable data structures since they're
   // easier to reason about and React works very well with them. That's why
   // we use map() and filter() everywhere instead of mutating the array or
   // todo items themselves.
-  this.todos = this.todos.map(function (todo) {
-	return Object.assign({}, todo, {completed: checked});
+  this.todos = this.todos.map(function(todo) {
+    return { ...todo, completed: checked };
   });
 
   this.inform();
 };
 
-app.TodoModel.prototype.toggle = function (todoToToggle) {
-  this.todos = this.todos.map(function (todo) {
-	return todo !== todoToToggle ?
-	  todo :
-	  Object.assign({}, todo, {completed: !todo.completed});
+app.TodoModel.prototype.toggle = function(todoToToggle) {
+  this.todos = this.todos.map(function(todo) {
+    return todo !== todoToToggle
+      ? todo
+      : { ...todo, completed: !todo.completed };
   });
 
   this.inform();
 };
 
-app.TodoModel.prototype.destroy = function (todo) {
-  this.todos = this.todos.filter(function (candidate) {
-	return candidate !== todo;
+app.TodoModel.prototype.destroy = function(todo) {
+  this.todos = this.todos.filter(function(candidate) {
+    return candidate !== todo;
   });
 
   this.inform();
 };
 
-app.TodoModel.prototype.save = function (todoToSave, text) {
-  this.todos = this.todos.map(function (todo) {
-	return todo !== todoToSave ? todo : Object.assign({}, todo, {title: text});
+app.TodoModel.prototype.save = function(todoToSave, text) {
+  this.todos = this.todos.map(function(todo) {
+    return todo !== todoToSave ? todo : { ...todo, title: text };
   });
 
   this.inform();
 };
 
-app.TodoModel.prototype.clearCompleted = function () {
-  this.todos = this.todos.filter(function (todo) {
-	return !todo.completed;
+app.TodoModel.prototype.clearCompleted = function() {
+  this.todos = this.todos.filter(function(todo) {
+    return !todo.completed;
```

Comment:
lol this might be it... revert these changes as it's a file that is tested.

it'll probably fix the test

Reply:
The fun thing is that yarn test runs 100% clean.

Reply:
yep... different suite. great work by the way

Reply:
it is so readable

Reply:
Thank you, this issue was 250% worth the time spent.